build.env

```diff
@@ -41,7 +41,9 @@ VM_DRIVER=none
 CHANGE_MINIKUBE_NONE_USER=true
 
 # Rook options
-ROOK_VERSION=v1.3.9
+ROOK_VERSION=v1.4.9
```

Comment:
Why use v1.4.9 instead of the latest v1.5.6?

Reply:
yeah as @travisn mentioned , its good to use latest version of Rook.

Reply:
That needs more changes, see #1721. It can be done in this PR, or an other one.

Reply:
I will raise a separate PR, first I just want snapshot retention to work in our e2e.

Reply:
Will take care of it asap